db/migrate/20190225133654_add_ignored_users_table.rb

```diff
@@ -0,0 +1,12 @@
+class AddIgnoredUsersTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :ignored_users, force: true do |t|
```

Comment:
Why the `force: true`?

Reply:
We definitely don't need it here, it will be dropping the table first before creating it. 😃